### PR TITLE
Tuna- merge_db : unused variables are removed

### PR DESCRIPTION
**Pull request by technicalgrp89**
Tuna - merge_db.py
Some of the unused variables are removed.

### CODE REVIEW

File: tests/test_utility.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+from tuna.utils.utility import get_mmi_env_vars
+
+LOGGER = setup_logger('utility')
+
+
+def test_utility():
+  test_arch2targetid
+  test_check_qts
+  test_get_env_vars
+  test_get_mmi_env_vars
```
Comment (cderb) [inline tests/test_utility.py:43]: unnecessary, pytest will call all functions in the file beginning with test_

File: tests/test_utility.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  assert (arch_value == f"{arch}:sram-ecc+:xnack-")
+
+
+def test_check_qts():
+
+  hostname = socket.gethostname()
+  m = Machine(hostname=hostname, local_machine=True)
+  retvalue = check_qts(hostname, logger=LOGGER)
+  print(retvalue)
+  assert (retvalue == False)
```
Comment (cderb) [inline tests/test_utility.py:69]: The outcome of this test depends on the machine it is ran on. This test won't be consistent across machines.

File: tests/test_utility.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  assert (retvalue == True)
+
+
+def test_get_env_vars():
+  env_vars = get_env_vars()
+
+  assert 'xyz_123' == (env_vars['user_name'])
+  assert 'xyz1234' == (env_vars['user_password'])
+  assert 'xyz.test.com' == (env_vars['db_hostname'])
+  assert 'testdb' == (env_vars['db_name'])
```
Comment (cderb) [inline tests/test_utility.py:91]: Where are the strings here inserted into the environment?
It doesn't look like this test does it.
Tests should be self contained.

File: tests/test_utility.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  hostname = socket.gethostname()
+  m = Machine(hostname=hostname, local_machine=True)
+  retvalue = check_qts(hostname, logger=LOGGER)
+  print(retvalue)
+  assert (retvalue == False)
+
+  m = Machine(hostname=hostname, local_machine=True)
+  retvalue = check_qts(hostname, logger=LOGGER)
+  print(retvalue)
+  assert (retvalue == False)
```
Comment (cderb) [inline tests/test_utility.py:74]: and here

File: tests/test_utility.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  assert 'xyz.test.com' == (env_vars['db_hostname'])
+  assert 'testdb' == (env_vars['db_name'])
+
+
+def test_get_mmi_env_vars():
+  env_vars = get_mmi_env_vars()
+
+  assert '10.100.00.000' == env_vars['gateway_ip']
+  assert '1234' == env_vars['gateway_port']
+  assert 'xyz' == env_vars['gateway_user']
```
Comment (cderb) [inline tests/test_utility.py:99]: and here

**Review by cderb — CHANGES_REQUESTED** [review]

**Comment by technicalgrp89** [comment]
duplicate entry. All are addressed in previous PR and merged #791

**Comment by technicalgrp89** [comment]
duplicate entry. All are addressed in previous PR and merged https://github.com/ROCmSoftwarePlatform/MITuna/pull/791

**Pull request closed without merging**